modules/power-monitoring-kepler-configuration.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/power-monitoring-configuration.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="kepler-configuration_{context}"]
+= The {PM-kepler} configuration
+
+You can configure {PM-kepler} with the `spec` field of the `kepler` resource.
+
+[IMPORTANT]
+====
+Ensure that the name of your {PM-kepler} instance is `kepler`. All other instances are ignored by the {PM-operator}.
+====
+
+The following is the list of configuration options:
+
+.{PM-kepler} configuration options
+|===
+|*Name* |*Spec* |*Description* |*Default* 
+|`port` |`exporter.deployment` |The port on the node where the Prometheus metrics are exposed. |`9103`
+|`nodeSelector` |`exporter.deployment` |The nodes on which {PM-kepler} exporter pods are scheduled. |`kubernetes.io/os: linux`
+|`tolerations` |`exporter.deployment` |The tolerations for {PM-kepler} exporter that allow the pods to be scheduled on nodes with specific characteristics. |`- operator: "Exists"`
+|===
+
+.Example `kepler` resource with default configuration
+[source, yaml]
+----
+apiVersion: kepler.system.sustainable.computing.io/v1alpha1
+kind: Kepler
+metadata:
+  name: kepler
+spec:
+  exporter:
+    deployment:
+      port: 9103 # <1>
```

Comment:
I got a peer review comment about this just recently with this link: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-asciidoc-callouts-in-the-yaml 
so that is why I use `#` in the YAML there. I hope that explains it. Thank you!